go.mod

```diff
@@ -39,8 +50,9 @@ require (
 	github.com/knative/build v0.3.1-0.20190330033454-38ace00371c7
 	github.com/knative/pkg v0.0.0-20190330034653-916205998db9
 	github.com/mattn/go-zglob v0.0.1
+	github.com/morikuni/aec v0.0.0-20170113033406-39771216ff4c // indirect
 	github.com/pelletier/go-toml v1.3.0
-	github.com/peterbourgon/diskv v0.0.0-20171120014656-2973218375c3
+	github.com/peterbourgon/diskv v2.0.1+incompatible
```

Comment:
I think changes to these dependencies have happened due to k8s.io/* version changes 😄